easy_thumbnails/management/commands/thumbnail_cleanup.py

```diff
@@ -35,15 +37,16 @@ def _check_if_exists(self, storage, path):
     def _delete_sources_by_id(self, ids):
         Source.objects.all().filter(id__in=ids).delete()
 
-    def clean_up(self, dry_run=False, verbosity=1, last_n_days=0,
-                 cleanup_path=None, storage=None):
+    def clean_up(
+        self, dry_run=False, verbosity=1, last_n_days=0, cleanup_path=None, storage=None
+    ):
         """
         Iterate through sources. Delete database references to sources
         not existing, including its corresponding thumbnails (files and
         database references).
         """
         if dry_run:
-            print ("Dry run...")
+            print("Dry run...")
```

Comment:
we should use `self.stdout.write(...)` here.

Reply:
I totally agree with this, but there are many such places. I would like to separate stylistic changes from functional ones.